loleaflet/src/control/Control.JSDialog.js

```diff
@@ -72,6 +73,13 @@ L.Control.JSDialog = L.Control.extend({
 		var content = L.DomUtil.create('div', 'lokdialog ui-dialog-content ui-widget-content', container);
 
 		var builder = new L.control.jsDialogBuilder({windowId: data.id, mobileWizard: this, map: this.map, cssClass: 'jsdialog'});
+
+		if (isModalPopup) {
+			var overlay = L.DomUtil.create('div', builder.options.cssClass + ' jsdialog-overlay ' + (data.cancellable ? 'cancellable' : ''), document.body);
+			if (data.cancellable)
+				overlay.onclick = function () { that.closePopover(data.id); };
```

Comment:
`closePopover` function is missing here (not ported from master)
it probably will need some adjustment to remove overlay from the body